cnv_somatic_pair_workflow.wdl

```diff
@@ -23,14 +23,25 @@
 # - The sites file (common_sites) should be a Picard or GATK-style interval list.  This is a list of sites
 #   of known variation at which allelic counts will be collected for use in modeling minor-allele fractions.
 #
+# - If you opt to run FuncotateSegments (i.e. set `is_run_funcotator` to `true`), then please also ensure that you have
+#       the correct value for `funcotator_ref_version`.  Treat `funcotator_ref_version` as required if
+#       `is_run_funcotator` is `true`.  Valid values for `funcotator_ref_version` are `hg38` and `hg19`.
+#       The latter includes GRCh37.
+#
+#
 # - Example invocation:
 #
 #       java -jar cromwell.jar run cnv_somatic_pair_workflow.wdl -i my_parameters.json
 #
 #############
 
-import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/1.3.0/cnv_common_tasks.wdl" as CNVTasks
-import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/1.3.0/cnv_somatic_oncotator_workflow.wdl" as CNVOncotator
+#import "../cnv_common_tasks.wdl" as CNVTasks
+#import "cnv_somatic_oncotator_workflow.wdl" as CNVOncotator
+#import "cnv_somatic_funcotate_seg_workflow.wdl" as CNVFuncotateSegments
+
+import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/dev/cnv_common_tasks.wdl" as CNVTasks
+import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/dev/cnv_somatic_oncotator_workflow.wdl" as CNVOncotator
+import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/dev/cnv_somatic_funcotate_seg_workflow.wdl" as CNVFuncotateSegments
```

Comment:
Reminder to update path to use next version of workflow release before merging to master